std/encoding/yaml/dumper/dumper.ts

```diff
@@ -0,0 +1,898 @@
+// Ported from js-yaml v3.13.1:
+// https://github.com/nodeca/js-yaml/commit/665aadda42349dcae869f12040d9b10ef18d12da
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Comment:
Is there a copyright line that should be stated from the port?

Reply:
I guess no because js-yaml licence is MIT

Reply:
Please reproduce their copyright line here.

Reply:
Will do 👌